boot/espressif/secureboot-sign-ec256.conf

```
@@ -7,6 +7,8 @@ CONFIG_SECURE_SIGNED_APPS_RSA_SCHEME=1
 CONFIG_SECURE_BOOT=1
 CONFIG_SECURE_BOOT_V2_ENABLED=1
 CONFIG_SECURE_BOOT_SUPPORTS_RSA=1
+CONFIG_SECURE_FLASH_ENC_ENABLED=y
+CONFIG_SECURE_FLASH_ENCRYPTION_MODE_DEVELOPMENT=y
```

```suggestion
CONFIG_SECURE_FLASH_ENCRYPTION_MODE_RELEASE=y
```
It is better for the CI to test a production-like configuration.

---

Done